tests/src/baseservices/varargs/varargsupport.il

```diff
@@ -1,114 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Should we keep these tests for Unix?

Reply:
Personally I do not see the value of the test and conditionally excluding it by platform will be more work to undo later. If there are strong opinions to keep them then I can add them back.

Reply:
This test would have caught the wrong ifdef that I have commented on...

Are we going to implement varargs support on Unix for next release? If yes, I do not have a problem with deleting these tests now. If not, we should keep them.

Reply:
Fair point will add them back thank you.